js-src/upload/index.ts

```diff
@@ -272,7 +277,7 @@ async function processFiles(files: TdrFile[]) {
 
             fileInfoList.push(fileInfo);
             filePathToFile[fileInfo.path!] = tdrFile;
-            fileInfoCount ++;
+            fileInfoCount++;
```

Comment:
Sorry about all this churn!

Do you think it's worth adding a linter now, or would that be more annoying than the churn? (We should definitely add one in Beta, but at that point we'll also have a test suite and other things for Jenkins to check.)

Reply:
I don't think its worth, given we aren't really doing that much on the prototype now.